cmd/bundle_create.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/snyk/policy-engine/pkg/bundle"
+	v1 "github.com/snyk/policy-engine/pkg/bundle/v1"
+	"github.com/spf13/cobra"
+)
+
+var (
+	bundleCreateOutput   string
+	bundleCreateRevision string
+	bundleCreateVCSType  string
+	bundleCreateVCSURI   string
+)
```

Comment:
```suggestion
var bundleCreateFlags struct {
	Output   string
	Revision string
	VCSType  string
	VCSURI   string
}
```

I guess like this would be a bit cleaner, but that's sort of a different refactoring

Reply:
I like this pattern! Done in cc8d406.